Packages/MIES/MIES_DataConfigurator.ipf

```diff
@@ -2376,9 +2378,15 @@ static Function DC_AddEpochsFromStimSetNote(panelTitle, channel, stimset, stimse
 
 						// active
 						subsubEpochBegin = subEpochBegin
-						// we never have a trailing baseline after the last pulse in the epoch
-						// to avoid decimation errors we assign all of the left over time to pulse active
-						subsubEpochEnd = (pulseNr == (flipping ? 0 : numPulses - 1)) ? subEpochEnd : (subEpochBegin + pulseDuration)
+
+						// normally we never have a trailing baseline with pulse train
+						// but if poisson distribution and flipping is combined we can have it
+						// to avoid decimation errors we assign all of the left over time to pulse active if we are not in this special case
+						if(!poissonDistribution && (pulseNr == (flipping ? 0 : numPulses - 1)))
```

Comment:
I think the if check somehow does not reflect the comment above as it does
subsubEpochEnd = subEpochBegin + pulseDuration
also for the (poissonDistribution + non-flipped) case.

Reply:
Thx. Fixed.